homework-submission/react-hyf/server/server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const app = express();
+app.use(express.static('../react-hyf-client/build'));
+
+app.get('/', async (request, response) => {
+  await response.sendFile('index.html', err => {
+    throw err;
```

Comment:
I have to mention two points here:
 1. You are throwing error by any case
 2. `app.use` is doing the exact same thing you are doing in `app.get` :)

Reply:
Many thanks for these advices ... I've tried them... I still have one question over the first point. How to use this err callback function of `sendFile` in this case?

The `app.get` is deleted as it's doing the same thing of `app.use`